09-typescript-in-react/README.md

```diff
@@ -0,0 +1,641 @@
+# TypeScript in React
+
+## Prerequisites
+
+Please go to [https://github.com/msd-code-academy](https://github.com/msd-code-academy)
```

Comment:
https://github.com/msd-code-academy/react-workshop and clone the repo

Reply:
Good catch, will fix 👍